torchvision/models/vision_transformer.py

```diff
@@ -148,12 +159,32 @@ def __init__(
         self.num_classes = num_classes
         self.representation_size = representation_size
         self.norm_layer = norm_layer
+        self.conv_stem_layers = conv_stem_layers
```

Comment:
Do we need to store this within the model and have it serialized? I don't think we need to use its info anywhere else.

Reply:
But the `_init_weights` method needs to know whether this parameter is None or not, do we pass it explicitly?